docs/ko/docs/history-design-future.md

```diff
@@ -0,0 +1,77 @@
+# History, Design and Future
+
+어느 날, [한 FastAPI 사용자](https://github.com/fastapi/fastapi/issues/3#issuecomment-454956920)가 이렇게 물었습니다:
+
+> 이 프로젝트의 역사가 궁금합니다. 몇 주 만에 멋진 결과를 낸 것 같아요. [...]
+
+여기 그 역사에 대해 간단히 설명합니다.
+
+---
+
+## 대안
+
+저는 여러 해 동안 머신러닝, 분산 시스템, 비동기 작업, NoSQL 데이터베이스 같은 복잡한 요구사항을 가진 API를 개발하며 여러 팀을 이끌어 왔습니다.
+
+이 과정에서 많은 대안을 조사하고, 테스트하며, 사용해야 했습니다. **FastAPI**의 역사는 그 이전에 나왔던 여러 도구의 역사와 밀접하게 연관되어 있습니다.
+
+[대안](alternatives.md){.internal-link target=_blank} 섹션에서 언급된 것처럼:
+
+> **FastAPI**는 이전에 나왔던 많은 도구들의 노력 없이는 존재하지 않았을 것입니다.
+>
+> 이전에 개발된 여러 도구들이 이 프로젝트에 영감을 주었습니다.
+>
+> 저는 오랫동안 새로운 프레임워크를 만드는 것을 피하고자 했습니다. **FastAPI**가 제공하는 기능들을 다양한 프레임워크와 플러그인, 도구들을 조합해 해결하려 했습니다.
+>
+> 하지만 결국에는 이 모든 기능을 통합하는 도구가 필요해졌습니다. 이전 도구들로부터 최고의 아이디어들을 모으고, 이를 최적의 방식으로 조합해야만 했습니다. 이는 :term:`Python 3.6+ 타입 힌트 <type hints>`와 같은, 이전에는 사용할 수 없었던 언어 기능이 가능했기 때문입니다.
+
+---
+
+## 조사
+
+여러 대안을 사용해 보며 다양한 도구에서 배운 점들을 모아 저와 개발팀에게 가장 적합한 방식을 찾았습니다.
+
+예를 들어, 표준 :term:`Python 타입 힌트 <type hints>`에 기반하는 것이 이상적이라는 점이 명확했습니다.
```

Comment:
Markdown isn't completed. you may check it in your documentation.(2)

Reply:
Same as the comment above!👆